### ISSUE TITLE
Admin calendar (date) heading should be announced when changed

### ISSUE THREAD

**Issue by stephenharris** [stephenharris/Event-Organiser#277]
Add `aria-live` to calendar heading.

**Comment by stephenharris** [comment]
See 253883b53763461ee0626acbc6ae467b070d288d
